code/game/objects/items/blackout.dm

```diff
@@ -0,0 +1,84 @@
+// Blackout tool, used to trigger massive electricity outttage on ship or station, including connected levels.
+// It may have additional shots to use, but currently balanced to one shot.
+
+/datum/uplink_item/item/tools/blackout
+	name = "High Pulse Electricity Outage Tool"
+	item_cost = 24
+	path = /obj/item/blackout
+	desc = "A device wich can create power virus in terminal, spread it in power network and temporally creating blackout."
```

Comment:
`A device which infects the power network with a virus via a local terminal, creating a temporary blackout.`